internal/command/helpers.go

```diff
@@ -129,32 +130,71 @@ func newStorageClient(psqlURI string) (storage.Storer, error) {
 }
 
 // mustGetPSQLURI returns if it's set the URI from the environment variable
-// envVarPSQLURL, otherwise if it's set the psql uri from the repository config,
-// if it's also not empty prints an error and exits.
+// [envVarPSQLURL], otherwise the psql uri from the repository
+// config, if both are empty, an error is printed and the application
+// terminated.
 func mustGetPSQLURI(cfg *cfg.Repository) string {
-	uri := getPSQLURI(cfg)
-	if uri == "" {
-		stderr.Printf("PostgreSQL connection information is missing.\n"+
-			"- set postgres_url in your repository config or\n"+
-			"- set the $%s environment variable", envVarPSQLURL)
-		exitFunc(exitCodeError)
+	if url := getPSQLURIEnv(); url != "" {
+		return url
 	}
 
-	return uri
+	if cfg.Database.PGSQLURL != "" {
+		return cfg.Database.PGSQLURL
+	}
+
+	stderr.ErrPrintln(ErrPSQLURIMissing)
+	exitFunc(exitCodeError)
+	return ""
 }
 
-func getPSQLURI(cfg *cfg.Repository) string {
-	if url := os.Getenv(envVarPSQLURL); url != "" {
-		return url
+func getPSQLURIEnv() string {
+	if envURI := os.Getenv(envVarPSQLURL); len(envURI) != 0 {
+		log.Debugf("using postgresql connection URL from $%s environment variable",
```

Comment:
Env variable references in documentation don't usually contain the dollar sign.
```suggestion
		log.Debugf("using postgresql connection URL from %s environment variable",
```

Reply:
fixed